### PR TITLE
Adding more samples

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR adds three more samples to the Office Scripts ref docs: `Range.clear`, `Workbook.getWorksheets`, and `Worksheet.setTabColor`.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b2b75d5](https://github.com/OfficeDev/office-scripts-docs-reference/commits/b2b75d57e8e673fc0cc57e900257f92ab2ea8596)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel/snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/generate-docs/json/excel/snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C16%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202007162054028484-53%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EJDr0ZHxqz96eCfY6QQVu1gBFqTzobmr4B6umYqVN3g%3D&st=2020-07-16T20%3A52%3A40Z&se=2020-08-16T20%3A57%3A40Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b7e1f975-26d5-fb7f-b635-d5c5bc9f9be0?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/code-snippets/excel-snippets.yaml [hunk 1]
```diff
@@ -381,3 +409,18 @@
...
+'ExcelScript.Worksheet#setTabColor:member(1)':
+  - |-
+    /**
+     * This script sets the tab color of every worksheet in the workbook to red.
+     */
+    function main(workbook: ExcelScript.Workbook) {
+      // Get all the worksheets in the workbook. 
+      let sheets = workbook.getWorksheets();
+      
+      // Set the tab color each worksheet to a random color
```
Comment (ElizabethSamuel-MSFT) [inline docs/code-snippets/excel-snippets.yaml:421]: Or `Set the tab color of each worksheet to red.` then you can delete the next comment (line 423).
```suggestion
      // Set the tab color of each worksheet to a random color.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4c2ed8d](https://github.com/OfficeDev/office-scripts-docs-reference/commits/4c2ed8d3588e84bbc943a05b3b10595beec24ae4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel/snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/generate-docs/json/excel/snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202008032307404303-53%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=AjJVOSjQxux7eNnhwxIk61%2FvgDEjsoMJgJaHqJggRns%3D&st=2020-08-03T23%3A06%3A01Z&se=2020-09-03T23%3A11%3A01Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b7e1f975-26d5-fb7f-b635-d5c5bc9f9be0?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[40cc910](https://github.com/OfficeDev/office-scripts-docs-reference/commits/40cc910ad44011e15959f08a2e73895ca19ef8b2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/excel-snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/code-snippets/excel-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.workbook.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml) | :white_check_mark:Succeeded | |
[generate-docs/json/excel/snippets.yaml](https://github.com/OfficeDev/office-scripts-docs-reference/blob/AlexJ-Samples/generate-docs/json/excel/snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C3%5Cb7e1f975-26d5-fb7f-b635-d5c5bc9f9be0%5CPullRequest%5C202008032309165707-53%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=u5Lc5lfK6Q1x7AIxXaJ6H2M1UEDppUaegBSyofuAS58%3D&st=2020-08-03T23%3A09%3A34Z&se=2020-09-03T23%3A14%3A34Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/b7e1f975-26d5-fb7f-b635-d5c5bc9f9be0?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**